Adyen/adyen-web
# Adding Playwright for E2E testing

## Summary

- Created new package within the repo which holds the Playwright E2E test project.
- Added E2E pipeline running for: Chrome, Firefox, Webkit



## Testing
1. Make sure that Vagrant isn't using port 3024
2. Install playwright in your machine by running: `npm install @playwright/test@1.30.0` , then `npx playwright install`
3. Go to `packages/e2e-playright/` folder and run `npm run test:headless` or `npm run test:headed`
